test/utils/generate_gtest_cmakelists.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import shutil
+import subprocess
+
+
+TEMPLATE = """# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Can you move this out into a template file? Like for pytorch they have https://github.com/pytorch/pytorch/blob/main/cmake/TorchConfig.cmake.in